docs/encyclopedia/nexus-use-cases.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: nexus-use-cases
+slug: /nexus/use-cases
+title: Common Use Cases for Temporal Nexus
+description: Common use cases for Temporal Nexus
+sidebar_label: Nexus Use Cases
+tags:
+  - Nexus
+keywords:
+  - nexus
+  - nexus use cases
+  - nemporal nexus onboarding
+  - setting up Nexus Endpoints
+  - temporal cloud
+  - nexus setup
+
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+
+:::
+
+[Temporal Nexus](/evaluate/nexus) enables the following use cases:
+
+- **Cross-team, cross-domain, and cross-namespace** \-
+connect Temporal Applications within and across Namespaces.
+- **Share a subset of a Temporal Application** \-
+abstract and share a subset of an Application as a Nexus Service.
+- **Modular design for growth** \-
+modular application design that can evolve as you grow.
+- **Smaller failure domains** \- 
+each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
+- **Multi-region** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+
+:::tip RELATED
+
+- [Evaluate](/evaluate/nexus) why you should use Nexus and learn more about [Nexus use cases](/evaluate/nexus#use-cases).
+
+:::
+
+## Share Workflows Across Namespaces
+
+Nexus is purpose-built to connect Temporal Applications within and across Namespaces.
+It addresses the limitations of Child Workflows, Activity Wrappers, and bespoke APIs that target a remote Namespace.
+Nexus has a streamlined Temporal developer experience, reliable execution, and integrated observability.
+
+In the past, when a caller Workflow invokes another Workflow directly the caller must know:
+- target Workflow's Namesapce and Task Queue.
+- target Workflow Policy options including [Workflow-Id-Reuse-Policy](/workflows#workflow-id-reuse-policy) and [Workflow-Id-Conflict-Policy](/workflows#workflow-id-conflict-policy).
```

Comment:
To me it's more about retry policy and timeouts, but yes also the ID reuse policy.

Reply:
Added those.